.goreleaser.yml

```diff
@@ -34,3 +36,77 @@ changelog:
     exclude:
       - '^docs:'
       - '^test:'
+
+# .goreleaser.yaml
+brews:
+  -
+    # Name template of the recipe
+    # Default to project name
+    name: hazelcast-commandline-client
+
+    tap:
+      owner: hazelcast
+      name: homebrew-hz
+
+      # Optionally a token can be provided, if it differs from the token provided to GoReleaser
+      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
+
+    # Template for the url which is determined by the given Token (github, gitlab or gitea)
+    #
+    # Default depends on the client.
+    url_template: "https://github.com/hazelcast/hazelcast-commandline-client/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+
+    # The project name and current git tag are used in the format string.
+    commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
+
+    # Caveats for the user of your binary.
+    # Default is empty.
+    caveats: "How to use this binary"
```

Comment:
Why do we have this in caveats section?

Reply:
Thanks, great catch. Missed that one https://github.com/hazelcast/hazelcast-commandline-client/pull/75/commits/6ae287545e1c0d7a0b81b524538dcfb9b9a1a4f3